CHANGELOG.md

```diff
@@ -730,6 +730,7 @@ replace github.com/confio/ics23/go => github.com/cosmos/cosmos-sdk/ics23/go v0.8
 * (upgrade) [#12603](https://github.com/cosmos/cosmos-sdk/pull/12603) feat: Move AppModule.BeginBlock and AppModule.EndBlock to extension interfaces
 * (telemetry) [#12405](https://github.com/cosmos/cosmos-sdk/pull/12405) Add *query* calls metric to telemetry.
 * (query) [#12253](https://github.com/cosmos/cosmos-sdk/pull/12253) Add `GenericFilteredPaginate` to the `query` package to improve UX.
+* (client) [#15463] Add status endpoint for clients
```

Comment:
This line does not look right (PR link missing).
Given that it is a `feat!` it would be nice to have an API breaking Changes line as well, mentioning this extra arg)

Reply:
oh seems we removed the automatic link creation step in our release process. Can we add it back? save some time.

Reply:
I still have that script locally (https://github.com/cosmos/cosmos-sdk/blob/release/v0.45.x/scripts/linkify_changelog.py) but usually no one forgets, and is it better to have added directly as it makes it clickable on main. I am curious about the time saving of not adding it 😆 

I, as well, mean that the line is at the wrong place.

Reply:
done